api-schema/src/main/java/org/stellar/anchor/api/sep/sep6/GetWithdrawRequest.java

```diff
@@ -0,0 +1,40 @@
+package org.stellar.anchor.api.sep.sep6;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+/**
+ * The request body of the GET /withdraw endpoint.
+ *
+ * @see <a
+ *     href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#request-1">GET
+ *     /withdraw</a>
+ */
+@Builder
+@Data
+public class GetWithdrawRequest {
```

Comment:
I understand the SEP-6 uses `GET` method. But it is confusing in the service layer. Can we rename it to something less of a "GETTER" name?

Reply:
Sure, I can update them outside of this PR.